### DISCUSSION TITLE
WLED + SK6812 White not working?

### DISCUSSION THREAD

**Discussion by jimmyleet** [awawa-dev/HyperHDR#265] [Q&A]
I am running HyperHDR on a Raspberry PI and used WLED to control the LED strips. However what I have noticed is that HyperHDR isn't using the white LED on the SK6812. For example when I goto color calibration with Kodi and it shows the full white the end half of my strip is red because it's using RGB to create white and not using the white on the LED Strip.
Is there a solution to this? When I run normal Hyperion with the same WLED setup it uses the white LED fine only just now after switching to HyperHDR am I having issues with white.

**Comment by awawa-dev** [comment]
And with Hyperion you also used USB or wifi connection? Anyway protocol for communication in both cases is always RGB, its WLED job to enable white channel calculating it from provided RGB values.

**Comment by jimmyleet** [comment]
Exact same setup just flashed the sd card from Hyperion to Hyperhdr a few
hours ago.
Android box > hdmi splitter > usb grabber ms2109 to the raspberry pi.
Which then sends the signal to my LEDs running WLED.

I'm running the beta hyperhdr when I wake up tomorrow I'll go back to the
stable and see it I have the same problem.

**Reply by awawa-dev** [reply]
And how Rpi send the data to WLED? Still it's not HyperHDR issue but rather WLED configuration problem (white channel not enabled in WLED) or limitation for some protocol. We use only RGB values.

**Comment by jimmyleet** [comment]
Just using the wifi and the WLED protocol. The same setting i used in
hyperion but that worked fine with white LED. Hmm I'll check my WLED
settings and maybe try hyperion again see what changed i appreciate the
quick help mate and I'll update with anything I discover.

**Comment by awawa-dev** [comment]
Now I had some time and did the test.  

Latest v18.0.0.0beta2 installer from the HyperHDR release section.  
Installed WLED 0.13.1 for ESP8266.  

In WLED I only changed led number, led type set to sk6812 RGBW, disabled 'automatic brightness limiter'
In HyperHDR remote control tab set static color gray RGB(16, 16, 16). Pure white is too bright and I was unable to take pictures using phone. HyperHDR's WLED network protocol.  

The white LED is clearly powered on.
![IMG_20220505_173501](https://user-images.githubusercontent.com/69086569/166961901-d7c81be8-8098-4f61-a1df-bdc07c2ea35b.jpg)  

Then I set in WLED 'Auto-calculate white channel from RGB' from default DUAL to NONE. And as was expected white LED is disabled now.  

![IMG_20220505_173155](https://user-images.githubusercontent.com/69086569/166962027-93e5b310-64be-4b66-b066-0b13250634d5.jpg)  
  
BTW I tested also the adalight USB communication and for that protocol 'WLED Auto-calculate white channel' also works (in fact that was the most interesting part of that test for me).

**Comment by jimmyleet** [comment]
Really appreciate your help, turns out it's my power supply that is faulty and not pushing the whites. Guess it all happened at the same time as I switched to HyperHDR.....
